### ISSUE TITLE
Image quality percentage doesn't work and the cropped image size is bigger than the original

### ISSUE THREAD

**Issue by Garamani** [hhunaid/react-native-image-crop-tools#10]
Thank you for this Great Image Cropper,

Everything works fine except the image compression. NO matter what percentage I use, the output has the same size and it is bigger than the original image (more than twice).

The original file size: 458 KB
The cropped Image size: 1.2 MB

**Comment by hhunaid** [comment]
What platform? Android or iOS?

**Comment by Garamani** [comment]
> What platform? Android or iOS?

I tested on iOS

**Comment by hhunaid** [comment]
Unfortunately image quality in PNG images is not supported in iOS. You can try exporting the image as a JPG if your input image is also a JPG then there will be no loss of alpha channel.

**Comment by Garamani** [comment]
All the images I use are JPG from URIs of UNSPLASH API. The [react-native-image-crop-picker
](https://github.com/ivpusic/react-native-image-crop-picker) Compression that I'm using in my project Works fine. It seems the Compression Process of this module is not getting involved when cropped images are being saved.

**Comment by hhunaid** [comment]
Can you paste an example code. One you are using to crop the image.

**Comment by Garamani** [comment]
This is the part of code I'm using, There Might be something wrong with this code:
```
     <CropView
       sourceUrl={this.state.unsplashData[this.state.unsplashSelectedImageIndex].urls.regular}
       style={styles.cropView}
       ref={ref => {
          this.cropViewRef = ref;
       }}
       onImageCrop={res => {this.setState({uploadingImage: res}) }}
       keepAspectRatio
       aspectRatio={{width: 414, height: 896}}
     />
            
    <Button
       title='DONE'
       onPress={() => {
         this.cropViewRef.saveImage(true, 50);
       }}
    >
```

**Comment by hhunaid** [comment]
this.cropViewRef.saveImage(true, 50)

Pass false instead of true here

**Comment by Garamani** [comment]
> this.cropViewRef.saveImage(true, 50)

I changed it to false: this.cropViewRef.saveImage(false, 50)
same result: the image size is 1.2 MB (No compression)

**Comment by hhunaid** [comment]
That’s really strange. I’ll look into it over the weekend.

**Comment by Garamani** [comment]
Thank you for your quick responses, I look forward to hearing from you.

**Comment by MoeMamdouh** [comment]
Any updates? @hhunaid

**Comment by MoeMamdouh** [comment]
I have create PR fixed this issue #17

**Comment by hhunaid** [comment]
Thanks. Closing.
